README.md

```diff
@@ -81,6 +81,19 @@ If you receive a "No machine capacity available" error message when submitting a
 
 FGMachine automatically reloads the `projects.json` file when it is changed.
 
+#### GPU capacity support
+
+In order to handle projects, which require GPUs to perform a task, you need to add two parameters for each project in `projects.json` file:
+
+```json
+{
+  "gpu_capacity": "<gpu capacity needed (as a fraction of one GPU capacity (e.g. 0.5)>",
+  "gpu_command": "<option to pass to script to identify card number (e.g. -gpu)>",
+}
+```
+
+Note that gpu_capacity represents (the inverse of) instances of the program the FGMachine host system can run on one GPU; for example a machine with 4 GPUs, will be able to run 8 instances of the program with capacity 0.1 and gpu_capacity 0.5.
+
```

Comment:
Add the following paragraph:

`gpu_capacity` automatically assigns a GPU for experiments, which makes it easier to run batch experiments. For manual control, it is recommended to use a GPU flag as part of the experiment hyperparameters in the project schema.